lib/commands/configure/index.js

```diff
@@ -8,7 +8,7 @@ const chalk = require('chalk');
 
 const configure = async() => {
   if(await checkConfigExists()){
-    logger.log(chalk.red.bold('⚠️ Mergify is already configured. Configuring again will override the existing file.'));
+    logger.warn('⚠️ Mergify is already configured. Configuring again will override the existing file.');
```

Comment:
Since all warnings have a ⚠️ in front maybe also put those in the logger. 
Same goes for the Uh-oh cat 🙀.

Reply:
Fixed